.pre-commit-config.yaml

```diff
@@ -24,8 +24,10 @@ repos:
     - id: watermark
       name: Check notebooks have watermark
       types: [jupyter]
-      entry: python scripts/check_watermark.py
-      language: python
+      entry: '%load_ext watermark.*%watermark -n -u -v -iv -w'
+      language: pygrep
+      args: [--negate, --multiline]
+      minimum_pre_commit_version: 2.8.0
```

Comment:
The exception message in `check_watermark.py` had a link to the Wiki page (https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style)
Is there a way to report this link here? Or at least append a note like ` (see Wiki)` to the name of the job, so users understand what this check is about.

Reply:
That's a really good point, thanks!